front-end/src/App.jsx

```diff
@@ -1,7 +1,8 @@
 import { useState } from "react";
 import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
-import "./App.css";
+//commenting out stylesheet since we're not using it and doing this makes it "full screen"
+// import "./App.css";
```

Comment:
If its not being used then it can be safely removed

Reply:
Falilou said he would remove it, but I've only talked about it with him, and if he did remove it his changes aren't in main yet. I want to make sure that's ok.

Reply:
Sounds good @johnmdrake1 lets keep it for now